test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/IncorpIdControllerSpec.scala

```diff
@@ -300,6 +312,16 @@ class IncorpIdControllerSpec extends ControllerSpec {
         verify(mockErrorPage).apply()(any[Request[_]](), any[Messages]())
       }
     }
+
+    "show verification error page" when {
```

Comment:
More of a question here but are other variations on this scenario possible? e.g. business verification status is `FAIL` but registration status is **not** `REGISTRATION_NOT_CALLED`, business verification status is `PASS` but registration status is `REGISTRATION_NOT_CALLED`, etc.?

If so, can we add some more tests around these in here please?

Reply:
As per the logic in incorporated-entity-identification-frontend. If business verification status is FAIL we will always get registration status REGISTRATION_NOT_CALLED. If business verification status is PASS and it fails for any other reason then it will return REGISTTRATION_FAILED.

Reply:
There is one more scenario listed in the where business verification status is 'UNCHALLENGED' and registration status is 'REGISTRATION_NOT_CALLED'  but I think it would be a separate scenario which is not handled yet or there is no story to handle this scenario.